src/module.js

```diff
@@ -50,8 +50,8 @@ function module_import() {
   return v.import.apply(v, arguments);
 }
 
-function module_variable(observer) {
-  return new Variable(TYPE_NORMAL, this, observer);
+function module_variable(observer, options = {}) {
```

Comment:
Remove the default here (one less object, and also if we did want a default, better to consolidate it on the Variable constructor rather than the *module*.variable method)?

```suggestion
function module_variable(observer, options) {
```